src/dj/ui/FPM.ftd

```diff
@@ -0,0 +1,6 @@
+-- import: fpm
+
+-- fpm.package: ui
+base: this is base
+
+-- fpm.dependency: fifthtry.github.io/fpm-controller-ui as ft
```

Comment:
Probably we should not call this variable `ft` as `ft` (and `ds`) as common aliases for `doc-site`. If I write `ft.code` assuming this `ft` like other `ft`s, and it would be tricky to debug. Not too tricky but still.